### PR TITLE
[PL-48750] PL 1.33.x Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-48750](https://harness.atlassian.net/browse/PL-48750)

https://harness.atlassian.net/wiki/spaces/PD/pages/21724594239/NG+Release+SAAS+April+1st+2024+for+1.32.x-+Build

https://harness.atlassian.net/wiki/spaces/PD/pages/21734916118/NG+Release+SAAS+April+5th+2024+for+1.33.x-+Build

For reviewers, a preview is available:
https://661e990773bf9134b0b434c6--harness-developer.netlify.app/release-notes/platform#version-1335

## Merge requirements

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

[PL-48750]: https://harness.atlassian.net/browse/PL-48750?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -80,6 +80,32 @@ The following deprecated API endpoints are longer supported:
...
+
+- AWS Secret Manager Connector will support both Plain Text & Encrypted type Access Key Id. (PL-48420)
+
+#### Fixed issues
+
+- Email steps were failing due to the system's inability to find users with emails containing uppercase letters, as Harness converts and stores all emails in lowercase. (PL-48532, ZD-60291)
+
+- Reset password emails were being sent from the "CG" system for all accounts, causing issues for accounts that are exclusively "NG". Now, if an account is identified as "NG" only, the reset password emails will be dispatched from the "NG" system, rectifying the issue for "NG" only accounts. (PL-38735)
+
+- Slack channel notifications failed due to an error related to explicitly setting the "Host" header as "hooks.slack.com". We have removed the explicit "Host" header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue in this version. (PL-47914)
```
Comment (brian-f-harness) [inline release-notes/platform.md:95]:
```suggestion
- Slack channel notifications failed due to an error related to explicitly setting the "Host" header as "hooks.slack.com". We have removed the explicit "Host" header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue in this version. This item requires Harness Delegate version 24.04.82703. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-47914)
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -80,6 +80,32 @@ The following deprecated API endpoints are longer supported:
...
+
+- In SCIM, creating a new user with special characters in their name failed, preventing the user from being added to Harness and resulting in discrepancies in user group membership between the Identity Provider and Harness. (PL-47614)
```
Comment (brian-f-harness) [inline release-notes/platform.md:97]:
```suggestion
- In SCIM, creating a new user with special characters in their name failed, preventing the user from being added to Harness and resulting in discrepancies in user group membership between the Identity Provider and Harness. This item requires Harness Delegate version 24.04.82703. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).  (PL-47614)
```

**Comment by brian-f-harness** [comment]
@saifeemustafaqharness Suggest following up on two items before review:
https://harness.atlassian.net/browse/PL-46947 - check with Bharat for RN requirements
https://harness.atlassian.net/browse/PL-48935?focusedCommentId=710519 - check with Marko for RN requirements

File: release-notes/platform.md [hunk 3]
```diff
@@ -80,6 +80,32 @@ The following deprecated API endpoints are longer supported:
 
 ## April 2024
 
+### Version 1.33.x<!--  April 15, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/platform.md:83]:
```suggestion
### Version 1.33.5<!--  April 15, 2024 -->
```

Comment (brian-f-harness) [reply release-notes/platform.md:83]: Please add https://harness.atlassian.net/browse/PL-48781

File: release-notes/platform.md [hunk 4]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+- AWS Secret Manager Connector will support both Plain Text & Encrypted type Access Key Id. (PL-48420)
+
+#### Fixed issues
+
+- Email steps were failing due to the system's inability to find users with emails containing uppercase letters, as Harness converts and stores all emails in lowercase. (PL-48532, ZD-60291)
+
+- Reset password emails were being sent from the "CG" system for all accounts, causing issues for accounts that are exclusively "NG". Now, if an account is identified as "NG" only, the reset password emails will be dispatched from the "NG" system, rectifying the issue for "NG" only accounts. (PL-38735)
+
+- Slack channel notifications failed due to an error related to explicitly setting the "Host" header as "hooks.slack.com". We have removed the explicit "Host" header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue in this version. This item requires Harness Delegate version 24.04.82703. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-47914)
```
Comment (brian-f-harness) [inline release-notes/platform.md:95]:
```suggestion
- Slack channel notifications failed due to an error related to explicitly setting the "Host" header as "hooks.slack.com". We have removed the explicit "Host" header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue in this version. This item requires Harness Delegate version 24.04.82707. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-47914)
```

File: release-notes/platform.md [hunk 5]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+- Delegate utilization metrics failed to decrease below a set threshold, even when rejecting all tasks. To solve this, memory-based threshold checks have been removed from the delegate due to functional discrepancies. This item requires Harness Delegate version 24.04.82703. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-48781, ZD-60713)
```
Comment (brian-f-harness) [inline release-notes/platform.md:97]:
```suggestion
- Delegate utilization metrics failed to decrease below a set threshold, even when rejecting all tasks. To solve this, memory-based threshold checks have been removed from the delegate due to functional discrepancies. This item requires Harness Delegate version 24.04.82707. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-48781, ZD-60713)
```

File: release-notes/platform.md [hunk 6]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+- In SCIM, creating a new user with special characters in their name failed, preventing the user from being added to Harness and resulting in discrepancies in user group membership between the Identity Provider and Harness. This item requires Harness Delegate version 24.04.82703. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).  (PL-47614)
```
Comment (brian-f-harness) [inline release-notes/platform.md:99]:
```suggestion
- In SCIM, creating a new user with special characters in their name failed, preventing the user from being added to Harness and resulting in discrepancies in user group membership between the Identity Provider and Harness. This item requires Harness Delegate version 24.04.82707. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).  (PL-47614)
```

File: release-notes/platform.md [hunk 7]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+The audit trail filter did not include an option for "Token" resource type, hindering the ability to filter for API token updates. Added a "Token" filter option to the audit trail, enabling users to specifically filter for Token & API Key updates through a dropdown menu. (PL-48606)
```
Comment (brian-f-harness) [inline release-notes/platform.md:101]:
```suggestion
The audit trail filter did not include an option for the **Token** resource type, hindering the ability to filter for API token updates. Added a **Token** filter option to the audit trail, enabling users to specifically filter for Token & API Key updates through a dropdown menu. (PL-48606)
```

File: release-notes/platform.md [hunk 8]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+- Users experienced authentication issues when switching the OAuth type from SAML, leading to incorrect OAuth authentication status displays. Implemented changes to accurately reflect the status (enabled or disabled) of OAuth authentication for users. (PL-48788, ZD-60869)
+
+- The feature flag subscription indicated it was expired, and the license was not updating automatically. Users had to manually pay each invoice after the billing cycle completed. Users can now view the correct billing due date on their subscription page. Additionally, the system has been updated to automatically charge the credit card on file when an invoice is generated. (PL-48671, ZD-57836)
+
+- The delegate task rejection metric was designed to reflect tasks rejected by a delegate due to system-related reasons (such as lack of resources or exceeding the limit of parallel tasks) but did not include specific details like task type or task ID. Enhanced the task rejection metrics by adding taskType and taskId labels. (PL-48488)
```
Comment (brian-f-harness) [inline release-notes/platform.md:107]:
```suggestion
- The delegate task rejection metric was designed to reflect tasks rejected by a delegate due to system-related reasons (such as lack of resources or exceeding the limit of parallel tasks) but did not include specific details like task type or task ID. Enhanced the task rejection metrics by adding `taskType` and `taskId` labels. (PL-48488)
```

File: release-notes/platform.md [hunk 9]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+- Users were being logged out when testing a Git connector with invalid credentials due to the Git client's "401" response being propagated to the UI. Implemented error handling to convert a "401" response from the test connection step to a "400", while preserving the original error message, preventing unintended user logouts. (PL-47753, ZD-58629)
```
Comment (brian-f-harness) [inline release-notes/platform.md:109]:
```suggestion
- Users were being logged out when testing a Git connector with invalid credentials due to the Git client's 401 response being propagated to the UI. Implemented error handling to convert a 401 response from the test connection step to a 400, while preserving the original error message, preventing unintended user logouts. (PL-47753, ZD-58629)
```

File: release-notes/platform.md [hunk 10]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
 
 ## April 2024
 
+### Version 1.33.5<!--  April 16, 2024 -->
+
+#### New features and enhancements
+
+- AWS Secret Manager Connector will support both Plain Text & Encrypted type Access Key Id. (PL-48420)
```
Comment (brian-f-harness) [inline release-notes/platform.md:87]:
```suggestion
- AWS Secret Manager connectors now support both plain text and encrypted type Access Key Ids. (PL-48420)
```

File: release-notes/platform.md [hunk 11]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+#### Fixed issues
+
+- Email steps were failing due to the system's inability to find users with emails containing uppercase letters, as Harness converts and stores all emails in lowercase. (PL-48532, ZD-60291)
+
+- Reset password emails were being sent from the "CG" system for all accounts, causing issues for accounts that are exclusively "NG". Now, if an account is identified as "NG" only, the reset password emails will be dispatched from the "NG" system, rectifying the issue for "NG" only accounts. (PL-38735)
```
Comment (brian-f-harness) [inline release-notes/platform.md:93]:
```suggestion
- Reset password emails were being sent from the FirstGen system for all accounts, causing issues for accounts that are exclusively NextGen. Now, if an account is identified as NextGen only, the reset password emails are dispatched from the NextGen system, rectifying the issue for NextGen only accounts. (PL-38735)
```

File: release-notes/platform.md [hunk 12]
```diff
@@ -80,6 +80,34 @@ The following deprecated API endpoints are longer supported:
...
+
+The audit trail filter did not include an option for "Token" resource type, hindering the ability to filter for API token updates. Added a "Token" filter option to the audit trail, enabling users to specifically filter for Token & API Key updates through a dropdown menu. (PL-48606)
+
+- Users experienced authentication issues when switching the OAuth type from SAML, leading to incorrect OAuth authentication status displays. Implemented changes to accurately reflect the status (enabled or disabled) of OAuth authentication for users. (PL-48788, ZD-60869)
+
+- The feature flag subscription indicated it was expired, and the license was not updating automatically. Users had to manually pay each invoice after the billing cycle completed. Users can now view the correct billing due date on their subscription page. Additionally, the system has been updated to automatically charge the credit card on file when an invoice is generated. (PL-48671, ZD-57836)
```
Comment (brian-f-harness) [inline release-notes/platform.md:105]:
```suggestion
- The feature flag subscription indicated it was expired, and the license was not updating automatically. Users had to manually pay each invoice after the billing cycle was completed. Users can now view the correct billing due date on their subscription page. Additionally, the system has been updated to automatically charge the credit card on file when an invoice is generated. (PL-48671, ZD-57836)
```

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**